### ISSUE TITLE
Don't show the words "Zoom in to edit" if we are about to be auto-zoomed

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/iD#8692]
1. Click https://www.openstreetmap.org/edit?gpx=3755308#map=14/24.18254969/120.86595668
1. Notice the words "Zoom in to edit", but then all the sudden we are taken to https://www.openstreetmap.org/edit?gpx=3755308#map=16/24.1825/120.8660

OK, then don't show us the words "Zoom in to edit".

**Comment by jidanni** [comment]
Also it should use smarter zooming. E,g., for this particular trace, at least level 19 is needed:
https://www.openstreetmap.org/edit?gpx=3755308#map=19/24.18209/120.86623

**Comment by jidanni** [comment]
Indeed, for us dark mode users there is an additional white flash present too.

**Comment by tyrasd** [comment]
confirming that for a short time the message appears while the map is already zoomed in to zoom level 16, where one does not need to zoom in further.

**Comment by tyrasd** [comment]
Today cannot reproduce it anymore. It could be that there the small changes we did in the meantime to the initialization of the editor fixed this.
